launch_app.py

```
@@ -0,0 +1,93 @@
+        return False
+
+def launch_app():
+    """Launch the Gradio application"""
+    print("🚀 Launching Wakeword Training Gradio Application...")
+
+    try:
+        # Change to the directory containing the app
+        os.chdir(os.path.dirname(os.path.abspath(__file__)))
+
+        # Launch the app
+        subprocess.run([sys.executable, "wakeword_training_gradio.py"])
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)